### PR TITLE
Users/mizanurrahman13/infra initialize membership project

### PR DESCRIPTION
**Pull request by mizanurrahman13**
Added JwtToken provider and separated default identity entities in membership project

### CODE REVIEW

File: Vazaar.Core.Api/Controllers/TokenController.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                }
+                else
+                {
+                    return BadRequest("Invalid credentials");
+                }
+            }
+            else
+            {
+                return BadRequest();
+            }
```
Comment (hassanhabib) [inline Vazaar.Core.Api/Controllers/TokenController.cs]: We can't have business logic in the controller's endpoints

File: Vazaar.Core.Api/Controllers/TokenController.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+namespace Vazaar.Core.Api.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    public class TokenController : ControllerBase
+    {
+        private readonly IConfiguration _configuration;
+        private readonly SignInManager<ApplicationUser> _signInManager;
+        private readonly UserManager<ApplicationUser> _userManager;
```
Comment (hassanhabib) [inline Vazaar.Core.Api/Controllers/TokenController.cs]: Please declare private class members according to our C# coding standard:
https://github.com/hassanhabib/CSharpCodingStandard

File: Vazaar.Core.Api/Startup.cs [hunk 3]
```diff
@@ -4,26 +4,104 @@
...
+                .AddEntityFrameworkStores<ApplicationDbContext>()
+                .AddUserManager<UserManager>()
+                .AddRoleManager<RoleManager>()
+                .AddSignInManager<SignInManager>()
+                .AddDefaultUI()
+                .AddDefaultTokenProviders();
+
+            services.Configure<IdentityOptions>(options =>
+            {
+                // Password settings.
```
Comment (hassanhabib) [inline Vazaar.Core.Api/Startup.cs]: No comments in code

File: Vazaar.Core.Membership/Seeds/RoleSeed.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        internal static Role[] Roles
+        {
+            get
+            {
+                return new Role[]
+                {
+                    new Role{ Id = Guid.NewGuid(), Name = "Admin", NormalizedName = "ADMIN", ConcurrencyStamp =  DateTime.Now.Ticks.ToString()  },
+                    new Role{ Id = Guid.NewGuid(), Name = "Customer", NormalizedName = "CUSTOMER", ConcurrencyStamp =  DateTime.Now.AddMinutes(1).Ticks.ToString()  }
+                };
```
Comment (hassanhabib) [inline Vazaar.Core.Membership/Seeds/RoleSeed.cs]: These roles can live in some storage

File: Vazaar.Core.Membership/Services/SignInManager.cs [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+// ---------------------------------------------------------------
+
+using Vazaar.Core.Membership.Entities;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+
```
Comment (hassanhabib) [inline Vazaar.Core.Membership/Services/SignInManager.cs:14]: Remove extra line

**Comment by ElbekDeveloper** [comment]
@yasirthite  please close this due to inactivity

**Pull request closed without merging**